phpseclib/Math/BinaryField/Integer.php

```diff
@@ -187,10 +182,9 @@ private static function polynomialDivide($x, $y)
     /**
      * Perform polynomial multiplation in the traditional way
      *
-     * @return string
      * @link https://en.wikipedia.org/wiki/Finite_field_arithmetic#Multiplication
      */
-    private static function regularPolynomialMultiply($x, $y)
+    private static function regularPolynomialMultiply($x, $y): string
```

Comment:
Should be able to do `string $x, string $y`

Reply:
Done